### PR TITLE
Post on the release of Cake for VSCode 2.0.0

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: input/blog/2021-12-14-cake-vscode-2.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Cake for Visual Studio Code 2.0.0 released
+category: Release Notes
+author: gep13
+---
+
+Version 2.0.0 of [Cake for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode) has been released.
+
+Highlights of this release include:
+- **Breaking change:** Cake for Visual Studio Code dropped support for deprecated Cake Runners
```
Comment (pascalberger) [inline input/blog/2021-12-14-cake-vscode-2.0.0-released.md:10]: We can add a link here to the blog post explaining which runners are deprecated

```suggestion
- **Breaking change:** Cake for Visual Studio Code 2.0 dropped support for [deprecated Cake Runners](/blog/2021/11/cake-v2.0.0-released#supported-runners-and-required.net-version)
```

**Review by pascalberger — APPROVED** [review]
One small suggestion, otherwise LGTM

File: input/blog/2021-12-14-cake-vscode-2.0.0-released.md (hunk 1 above)
Comment (gep13) [reply input/blog/2021-12-14-cake-vscode-2.0.0-released.md]: @pascalberger good idea!  I have updated the PR with this suggestion.

**Review by nils-a — APPROVED** [review]
LGTM!

**Pull request merged**